configs/sst_security_special_projects-keylime.yaml

```
@@ -8,23 +8,6 @@ data:
   packages:
     - keylime
     - keylime-agent-rust
```

@bstinsonmhk has agreed that the dependencies which are unique to keylime, and therefore by definition are being maintained by the same SST, should continue to be tracked here.
```suggestion
    - keylime-agent-rust
    - python3-alembic
    - python3-lark
    - python3-sqlalchemy
    - python3-tornado
```

---

OK, done